src/containers/Home.js

```diff
@@ -1,29 +1,21 @@
 /*eslint no-unused-vars:0, no-console:0 */
-import Header from 'twreporter-react-index-page-components/lib/components/header'
-import EditorPicks from 'twreporter-react-index-page-components/lib/components/editor-picks'
 import Helmet from 'react-helmet'
-import InfographicSection from 'twreporter-react-index-page-components/lib/components/infographic-section'
-import Latest from 'twreporter-react-index-page-components/lib/components/latest'
-import LatestTopic from 'twreporter-react-index-page-components/lib/components/latest-topic'
 import React from 'react'
-import Reviews from 'twreporter-react-index-page-components/lib/components/reviews'
-import CategorySection from 'twreporter-react-index-page-components/lib/components/category-section'
-import PhotographySection from 'twreporter-react-index-page-components/lib/components/photography-section'
-import ReporterIntro from 'twreporter-react-index-page-components/lib/components/reporter-intro'
-import SideBar, { moduleIdObj } from 'twreporter-react-index-page-components/lib/components/side-bar'
-import TopicsSection from 'twreporter-react-index-page-components/lib/components/topics-section'
 import categoryListID from '../conf/category-list-id'
 import categoryString from '../constants/category-strings'
 import categoryURI from '../conf/category-uri'
 import clone from 'lodash/clone'
 import get from 'lodash/get'
+import IndexPageComposite from 'twreporter-react-index-page-components/lib'
```

Comment:
just `import IndexPageComposite from 'twreporter-react-index-page-components`.
'twreporter-react-index-page-components/lib/index.js` is designed as `index` in package.json